kafka-connect/build.gradle

```diff
@@ -197,8 +197,8 @@ project(':iceberg-kafka-connect:iceberg-kafka-connect-runtime') {
           from configurations.runtimeClasspath
         }
         into('doc/') {
-          from "$projectDir/LICENSE"
-          from "$projectDir/NOTICE"
+          from "$projectDir/main/LICENSE"
+          from "$projectDir/main/NOTICE"
```

Comment:
it looks like the `main` folder doesn't exist on 1.7.x: https://github.com/apache/iceberg/tree/1.7.x/kafka-connect/kafka-connect-runtime